packages/toolpad-app/src/toolpad/ToolpadShell/ToolpadNavigation.tsx

```diff
@@ -8,16 +8,17 @@ import {
   Drawer as MuiDrawer,
   List,
   Stack,
-  Typography,
   IconButton,
+  Tooltip,
 } from '@mui/material';
 import { Link, matchPath, useLocation } from 'react-router-dom';
 
-const DRAWER_WIDTH = 68; // px
+const DRAWER_WIDTH = 44; // px
```

Comment:
The sidebar was too bulky and a bit ugly, I think we can keep it smaller and simpler.

Reply:
Could always make it resizable :)